lms/djangoapps/instructor/views/api.py

```diff
@@ -2295,10 +2296,9 @@ def list_report_downloads(_request, course_id):
     """
     course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
     report_store = ReportStore.from_config(config_name='GRADES_DOWNLOAD')
-
     response_payload = {
         'downloads': [
-            dict(name=name, url=url, link='<a href="{}">{}</a>'.format(url, name))
+            dict(name=name, url=url, link=HTML('<a href="{}">{}</a>').format(url, name))
```

Comment:
@robrap What is the best way to fix [`python-wrap-html`](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html?highlight=xss#python-wrap-html) violation.

I have fixed this one using HTML wrapper, but this seems failing the test which can't find the JSON.